### DISCUSSION TITLE
Can't find Snowfall theme

### DISCUSSION THREAD

**Discussion by swameher** [rocksdanister/lively#2511] [Ideas]
I've been searching for the snowfall theme shown on the website, but after many searches, I still couldn't figure out where to find it.
![image](https://github.com/user-attachments/assets/264d16b0-48a0-495c-8a4b-0045357cbf29)

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/snow
https://github.com/rocksdanister/snow/releases
